src/command.ts

```diff
@@ -97,6 +99,10 @@ export default abstract class Command {
     return this.constructor as typeof Command
   }
 
+  get isBeingRunByDefault() {
+    return !!this.options && !!this.options.isBeingRunByDefault
```

Comment:
Why is `isBeingRunByDefault` needed? If you really need different functionality, an extended command might be better.

Reply:
In my use case, my command is both a normal command and a 'default' command. I need to know when it is run because the user typed `<bin> <command>` and when it is run because the user typed `<bin>`. Passing that info to the command seems like an elegant solution to me.

I'm nudging for 'default command' to be a first class citizen in the world of oclif commands instead of a set of plugins, partly because it seems like an inherently useful feature of the library and partly because it requires changes to how the Main command currently works (how it handles the case where no args are passed to the command) and how the plugin-help works (needs changes to correctly and clearly document the default command usage).